ckanext/scheming/templates/scheming/form_snippets/org_to_allow_transfer_to.html

```diff
@@ -0,0 +1,80 @@
+{% import 'macros/form.html' as form %}
+
+{% macro render_field() -%}
+  <style>
+    #transfer_dataset_collapse .well{
+      margin: 0;
+    }
+    #transfer_dataset_collapse ul{
+      margin: 0;
+    }
+  </style>
+
+  {%- set options = [{
+    'value': '',
+    'text': _('No organisation')
+  }] -%}
+  {% for organization in h.get_all_organizations() %}
+    {% if organization.id != data.organization.id %}
+      {%- do options.append({
+        'value': organization.id,
+        'text': organization.display_name
+      }) -%}
+    {% endif %}
+  {% endfor %}
+
+  <div>
+    <a
+      class="btn btn-default"
+      data-toggle="collapse"
+      href="#transfer_dataset_collapse"
+      role="button"
+      aria-expanded="false"
+      aria-controls="transfer_dataset_collapse"
+    >
+      <span>{{_('Transfer Dataset')}}</span>
+    </a>
+  </div>
+
+  <br>
+
+  <div class="panel panel-default collapse" id="transfer_dataset_collapse">      
+      <div class="panel-body" style="margin: 0;">
+        <div class="row">
+            <div class="col-md-6">
+              <div class="well">
+                <p>{{_('How dataset transfers work:')}}</p>
+                <ul>
+                  {% for bullet_point in [
+                    'This tool allows you to transfer ownership of this dataset to another organisation for which you do not have data editing rights.',
+                    'Once an organisation is set, admins of the desired organisation will be able to see a transfer button when viewing this dataset.',
+                    'Clicking the transfer button will move the dataset into the desired organization and they will gain full control over it.'
+                    ] %}
+                      <li>{{ _(bullet_point) }}</li>
+                  {% endfor %}
+                </ul>
+              </div>
+            </div>
+            <div class="col-md-6">
+              {% call form.select(
+                field.field_name,
+                id='field-' + field.field_name,
+                label=h.scheming_language_text(field.label),
+                options=options,
+                selected=data.org_to_allow_transfer_to,
+                error=errors[field.field_name],
+                classes=['control-medium'],
+                attrs={'data-module':'autocomplete'},
+                is_required=h.scheming_field_required(field)
+                )
+              %}
+              {% endcall %}
+            </div>
+        </div>  
+      </div>
+  </div>  
+{%- endmacro %}
+
+{% if data.organization %}
```

Comment:
@jonathansberry can you confirm this is a correct way to go about solving for your notes on:

"I'd suggest hiding the transfer dataset field on create, and only show the field on edit/update (I think you can just check if the "data" object is populated from the jinja2 form snippet)"

Reply:
Also - it kinda sucks that jinja2 doesn't have an easy way to simply `return` which I could do at the very top 😅  rather than making a macro and then calling it based on the result of an if-statement

Reply:
Yes I think this is fine. Can you not wrap the whole template including the macro in the if statement? (I don't know for sure, just wondering - it's not a really big issue)